source/linear-algebra/source/02-EV/07.ptx

```diff
@@ -128,13 +128,73 @@ Rewrite this solution space in the form <me>\setBuilder{ a \left[\begin{array}{c
 </task>
 <task>
     <p>
-Rewrite this solution space in the form <me>\vspan\left\{\left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown\end{array}\right], \left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown \end{array}\right]\right\}.</me>
+Which of these choices best describes the set of two vectors
+<m>\left\{\left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown\end{array}\right], \left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown \end{array}\right]\right\}</m>
+used in this span?
+<ol marker="A.">
+<li>
+    <p>
+    The set is linearly dependent.
+    </p>
+  </li>
+<li>
+    <p>
+        The set is linearly independent.
+    </p>
+  </li>
+  <li>
+      <p>
+        The set spans the solution space.
+      </p>
+    </li>
+<li>
+    <p>
+        The set fails to span the solution space.
+    </p>
+  </li>
+  <li>
+    <p>
+        The set is a basis of the solution space. 
+    </p>
+  </li>
+</ol>
+    </p>
+</task>
+</activity>
+
+<sage language="octave">
+</sage>
+<activity estimated-time='10'>
+    <introduction>
+        <p>
+Consider the homogeneous system of equations
+<md alignat-columns='5' alignment="alignat">
+    <mrow>
+2x_1&amp;\,+\,&amp;4x_2&amp;\,+\,&amp;2x_3 &amp;\,-\,&amp;3 x_4 &amp;\,+\,&amp;31x_5&amp;\,+\,&amp;2x_6&amp;\,-\,&amp;16x_7&amp;=&amp; 0
+</mrow>
+<mrow>
+-1x_1&amp;\,-\,&amp;2x_2&amp;\,+\,&amp;4x_3 &amp;\,-\,&amp;x_4 &amp;\,+\,&amp;2x_5&amp;\,+\,&amp;9x_6&amp;\,+\,&amp;3x_7&amp;=&amp; 0
+</mrow>
+<mrow>
+x_1&amp;\,+\,&amp;2x_2&amp;\,+\,&amp;x_3 &amp;\,+\,&amp; x_4 &amp;\,+\,&amp;3x_5&amp;\,+\,&amp;6x_7&amp;\,+\,&amp;7x_7&amp;=&amp; 0
+    </mrow>
+</md>
+        </p>
+    </introduction>
+<task>
+    <p>
+Find its solution set (a subspace of <m>\IR^4</m>).
+    </p>
+</task>
```

Comment:
Since this is such a hefty example, can we just provide the solution set?

Reply:
in my experience, this is good practice. Each term I've had students make up their own 3x7 matrix and do the procedure to get practice. They almost always do it correctly and troubleshoot the errors, even with messy fractions and such. By giving them this specific example now, I'd be able to use the teams to troubleshoot since they'll all be getting the same answer.

Reply:
Sounds fine - if we have the answer in the instructor version, then an instructor can choose to spoil the solution set if they decide to move more quickly for pedagogical or time considerations.